src/main/java/com/exactpro/th2/common/schema/grpc/router/GrpcRouter.java

```diff
@@ -31,7 +29,10 @@
     /**
      * Initialization router
      */
-    void init(GrpcRouterConfiguration configuration);
+    @Deprecated(since = "3.9.0", forRemoval = true)
+    void init(GrpcConfiguration configuration);
+
+    void init(@NotNull GrpcConfiguration configuration, @NotNull GrpcRouterConfiguration routerConfiguration);
```

Comment:
Maybe switch to constructor from init method

Reply:
If we will do it, we should find a special constructor for the class. We can not be sure the constructor exists, but interface requires the developer to implement `init` method.